neo/Consensus/ConsensusContext.cs

```diff
@@ -205,17 +210,50 @@ private void SignPayload(ConsensusPayload payload)
             payload.Witness = sc.GetWitnesses()[0];
         }
 
+        /// <summary>
+        /// Prevent that block exceed the max size
+        /// </summary>
+        /// <param name="txs">Ordered transactions</param>
+        internal void EnsureMaxBlockSize(IEnumerable<Transaction> txs)
+        {
```

Comment:
Isnt this supposed to be part of filtering method? (Old SimplePolicy). It shouldnt be on consensus proposal I guess..

Reply:
You prefer this at the policy contract?

Reply:
I mean the SimplePolicyPlugin... we had some sort of FilterTxForBlock, that did the sorting and counting. I need to see where this is now... its the place to enforce a maxblock size limit I guess.

Reply:
This was moved in a recent PR, to Native contract Policy

Reply:
I prefer to move this to `ConsensusService`.

Reply:
`ConsensusService` has a circular dependency of `ConsensusContext`, for me is good as now, but feel free to change it